src/worker/helpers.go

```diff
@@ -193,7 +193,12 @@ func stopOL(_ string) error {
 	}
 	fmt.Printf("Send SIGINT and wait for worker to exit cleanly.\n")
 	if err := p.Signal(syscall.SIGINT); err != nil {
-		return fmt.Errorf("Failed to send SIGINT to PID %d (%s).  May require manual cleanup.\n", pid, err.Error())
+		// Failed to send SIGINT to PID  (os: process already finished).  May require manual cleanup will happen when system force shuts down.
+		if err.Error() == "os: process already finished" {
+			fmt.Printf("Unclean exit detected, trying automatic cleanup...")
+			return cleanupOL(olPath)
+		}
+		// return fmt.Errorf("Failed to send SIGINT to PID %d (%s).  May require manual cleanup.\n", pid, err.Error())
 	}
 
 	for i := 0; i < 600; i++ {
```

Comment:
perhaps sending signal 0 is a hacky way to see if the process has exited.  I wonder if there's any reason not to use FindProcess again?  Seems that would be cleaner.

Reply:
Perhaps before printing "OL worker process stopped successfully.\n" we should check that worker.pid was actually deleted.  Because if it's there, then the OL process might have crashed when attempting to stop.